src/ddl/covid_hosp.sql

```diff
@@ -0,0 +1,408 @@
+| inpatient_beds_utilization_numerator                             | int(11) | YES  |     | NULL    |                |
+| inpatient_beds_utilization_denominator                           | int(11) | YES  |     | NULL    |                |
+| percent_of_inpatients_with_covid                                 | double  | YES  |     | NULL    |                |
+| percent_of_inpatients_with_covid_coverage                        | int(11) | YES  |     | NULL    |                |
+| percent_of_inpatients_with_covid_numerator                       | int(11) | YES  |     | NULL    |                |
+| percent_of_inpatients_with_covid_denominator                     | int(11) | YES  |     | NULL    |                |
+| inpatient_bed_covid_utilization                                  | double  | YES  |     | NULL    |                |
+| inpatient_bed_covid_utilization_coverage                         | int(11) | YES  |     | NULL    |                |
+| inpatient_bed_covid_utilization_numerator                        | int(11) | YES  |     | NULL    |                |
+| inpatient_bed_covid_utilization_denominator                      | int(11) | YES  |     | NULL    |                |
+| adult_icu_bed_covid_utilization                                  | double  | YES  |     | NULL    |                |
+| adult_icu_bed_covid_utilization_coverage                         | int(11) | YES  |     | NULL    |                |
+| adult_icu_bed_covid_utilization_numerator                        | int(11) | YES  |     | NULL    |                |
+| adult_icu_bed_covid_utilization_denominator                      | int(11) | YES  |     | NULL    |                |
+| adult_icu_bed_utilization                                        | double  | YES  |     | NULL    |                |
+| adult_icu_bed_utilization_coverage                               | int(11) | YES  |     | NULL    |                |
+| adult_icu_bed_utilization_numerator                              | int(11) | YES  |     | NULL    |                |
+| adult_icu_bed_utilization_denominator                            | int(11) | YES  |     | NULL    |                |
++------------------------------------------------------------------+---------+------+-----+---------+----------------+
+
+- `id`
+  unique identifier for each record
+- `issue`
+  the day (YYYYMMDD) that the dataset was published
+- `date`
+  the day (YYYYMMDD) to which the data applies
+
+NOTE: Names have been shortened to 64 characters, as this is a technical
+limitation of the database. Affected names are:
+
+staffed_icu_adult_patients_confirmed_and_suspected_covid ->
+  staffed_icu_adult_patients_confirmed_suspected_covid
+staffed_icu_adult_patients_confirmed_and_suspected_covid_coverage ->
+  staffed_icu_adult_patients_confirmed_suspected_covid_coverage
+staffed_icu_adult_patients_confirmed_covid ->
+  staffed_icu_adult_patients_confirmed_covid
```

Comment:
these are identical

Reply:
wow nice catch. i double-checked both lists again. these two weren't changed, and none of the changed ones are missing from this list. updated to remove these.